### DISCUSSION TITLE
Значок, иконка ico для сайта-поиска

### DISCUSSION THREAD

**Discussion by audetv** [terratensor/svodd#74] [Идеи]
Давайте свои идеи, что можно изобразить для значка - иконки сайта, будет отображаться в браузере, так же можно поставить значок на сайт.
У меня была идея изобразить граф - математический объект. Сеть, нейросеть. Это конечно не совсем про поиск по комментариям на ФКТ, но планов у нас уже не на один год.

**Comment by iprst** [comment]
Мои варианты просты. Если можно, для проверки примерить, посмотрим как будет выглядеть. 

[favicon_A.zip](https://github.com/audetv/fct-search/files/11236696/favicon_A.zip)

<img src="https://user-images.githubusercontent.com/129882753/232149417-d2b41e20-d56d-4541-86a1-6bfbaab12675.png" width="100" height="100" />

[favicon_B.zip](https://github.com/audetv/fct-search/files/11236697/favicon_B.zip)

<img src="https://user-images.githubusercontent.com/129882753/232149423-68f2ceca-23f6-429f-90f9-468de3ac4524.png" width="100" height="100" />

[favicon_C.zip](https://github.com/audetv/fct-search/files/11236763/favicon_C.zip)

<img src="https://user-images.githubusercontent.com/129882753/232149425-29001286-2040-4d5e-a236-2928f7cc2c3d.png" width="100" height="100" />

Я за нижний вариант.

**Comment by audetv** [comment]
Добавил 3-ий вариант  #77 
Обновил. В браузере надо кэш сбросить, чтобы иконка обновилась: shift+F5

**Comment by audetv** [comment]
#80 доблены иконки различных размеров.
Для мобильных устройств, переходим: 
https://fct-search.audetv.ru 
Далее в настройках в браузера, обычно - три точки справа вверху окнах, выбрать, сохранить на главный экран, и появится значок звезды на экране мобильного рабочего стола. Работает в режиме Web приложения. Можно заходить на сайт по заначку, если кому так удобнее.

**Reply by iprst** [reply]
> появится значок звезды на экране мобильного рабочего стола

Ага, типа иконки приложения. Наверное на крупной иконке следует написать что-то опознавательное.
